lib/modules/manager/cocoapods/artifacts.spec.ts

```diff
@@ -205,7 +205,7 @@ describe('modules/manager/cocoapods/artifacts', () => {
     fs.getSiblingFileName.mockReturnValueOnce('Podfile.lock');
     fs.findLocalSiblingOrParent.mockResolvedValueOnce('Podfile.lock');
     fs.readLocalFile.mockResolvedValueOnce('Old Podfile.lock');
-    fs.outputFile.mockResolvedValueOnce(null as never);
+    fs.outputCacheFile.mockResolvedValueOnce(null as never);
```

Comment:
```suggestion
    fs.outputCacheFile.mockResolvedValueOnce();
```
should work for void function.